internal/store/store.go

```diff
@@ -556,8 +554,18 @@ func (s *Store) GetImageBuildImage(composeId uuid.UUID, imageBuildId int) (io.Re
 		return nil, 0, &NoLocalTargetError{"compose does not have local target"}
 	}
 
+	name, ok := imageBuild.Distro.ToString()
+	if !ok {
+		panic("distro name was validated earlier")
```

Comment:
I don't understand the error message here, what does it mean "validated earlier"?

Reply:
When creating the `Distro` object. It actually doesn't make sense to have the error here, for the same reason as above: we're 100% certain that this cannot fail.

Reply:
In this specific case, I'd prefer to change the `GetDistro` signature (same as above) and we can drop this check completely.

For the error message: It might just be my English skill but I'd be very confused if I saw this error message. I'd write something like: "internal error, distro variable contains non-existing value". In general something more specific and longer.

Reply:
You would never see this error message. I'll remove the check. It is pointless.